.taskcluster.yml

```diff
@@ -0,0 +1,27 @@
+version: 0
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - pull_request.opened
+          - pull_request.reopened
+          - pull_request.synchronize
+    payload:
+      maxRunTime: 7200
+      image: piatra/asmochitests
+      command:
+        - /bin/bash
+        - '--login'
+        - '-c'
+        - >-
+          git clone {{event.head.repo.url}} /activity-stream && cd /activity-stream &&
+          git checkout {{event.head.sha}} && npm install . && bash ./mochitest.sh
```

Comment:
why not add `npm install` to the bash script since that has the rest of your test-related setup?